includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,290 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor HTML widget.
+ *
+ * Elementor widget that insert a custom HTML code into the page.
+ *
+ * @since 1.0.0
+ */
+class Widget_Playing_Cards extends Widget_Base {
+
+	/**
+	 * Get widget name.
+	 *
+	 * Retrieve HTML widget name.
+	 *
+	 * @since 1.0.0
```

Comment:
```suggestion
	 * @since 3.18.0
```

Reply:
Update still missing here

Reply:
because i didnt upload all the fixes yet :)